### DISCUSSION TITLE
Lap history per driver

### DISCUSSION THREAD

**Discussion by diegocurti** [f1multiviewer/issue-tracker#362] [Feature requests]
## Description

In **Live Timing**, I think it would be useful if we can click somewhere on the driver's "row" to see a list or table of **lap history** with _time, sectors, tyre, tyre age at that moment_, etc.

Similar to clicking in the current tyre but with all information that I mentioned:

<details><summary>📸</summary>

![image](https://user-images.githubusercontent.com/1633756/225985830-a9f1344a-8e76-4823-904b-0f80fa0505d8.png)

</details>

Maybe clicking in the **Best lap** should open this feature, it's just an idea 😸 

Thanks in advance!

PS: Just for more information, this feature came into my mind while watching practices. I wanted to see all Leclerc's laps to see how degradation was going. I don't know if this would be useful in the race.

**Comment by Ankush-Pathak** [comment]
A graph of lap times overtime instead of a table I think would be easier to digest.
